UnityProject/Assets/Scripts/UI/Systems/Lobby/SongTracker.cs

```diff
@@ -163,7 +163,18 @@ private async Task PlayRandomTrack()
 
 		public void OnVolumeSliderChange()
 		{
-			MusicManager.Instance.ChangeVolume(volumeSlider.value);
+			if (volumeSlider.value == 0)
```

Comment:
no entirely sure but I imagine ``volumeSlider.value`` is a float, maybe it would be worth to compare it with some tolerance

Reply:
I think it makes only sense to mute the music when the volume slider is at 0 witch is all the way down. The number should also never go below 0 cause the slider is limited at 0.

Reply:
well yes, if the value is ``0.000000000096f`` it might be very low but not muted, but t hat isn't like it will break anything. I will accept it as is.

Reply:
I will test this tomorrow. But should not be de case 0.00096f should still be very silent. The Volume was only set to full blast when the variable was exactly 0. Of curse 0.00096f would not mute the music entirely but in this case there is still the Mute button. Will test tomorrow if 0.00096f breaks anything i will fix it. thx. 

Only that we understand each other: 0.1 is silent 1 is the loudest. Scale breaks at 0. 0 should be muted in my opinion.